src/app/dashboard/dashboard.component.html

```diff
@@ -1,14 +1,14 @@
 <div class="wraper">
-  <app-navbar [pageName]="dashboardPage"></app-navbar>
+  <app-navbar [pageName]="dashboardPage" (clicked)="openNewTripDialog()"></app-navbar>
   <div class="trip">
     <div class="trip-info" *ngFor="let trip of trips">
       <div class="trip-info__header">
-        <h1>{{trip.tripName}}</h1>
-        <h2>{{trip.tripDestination}}</h2>
+        <h1>{{trip.title}}</h1>
+        <h2>{{trip.destination}}</h2>
       </div>
       <div class="trip-info__dates">
-        <span>Start Date:{{trip.startDate}}</span>
-        <span>End Date:{{trip.endDate}}</span>
+        <span>Start Date:{{trip.startDate|date: 'shortDate'}}</span>
```

Comment:
Try to quickformat this line Cmd + Shift + P or Cmd + Shift + L
```suggestion
        <span>Start Date:{{trip.startDate | date: 'shortDate'}}</span>
```